### ISSUE TITLE
Incompatible with Admobs

### ISSUE THREAD

**Issue by Shinzah** [shprink/ionic-native-transitions#48]
I have an issue when using AdMobs and Native Transitions.
For the first page, all work fine.

![screenshot_2015-12-17-12-12-02](https://cloud.githubusercontent.com/assets/2910496/11868405/d17c4cd6-a4b7-11e5-9402-b1de6bc9340e.png)

But when i change page, the ad go to the top of the screen, the previous page is in the center of the page, and the new page is behind the previous page (crazy !!)
![screenshot_2015-12-17-12-12-20](https://cloud.githubusercontent.com/assets/2910496/11868408/d7b04396-a4b7-11e5-9e1f-c03fcc4b43f5.png)

**Comment by Shinzah** [comment]
Note that it only append when using AdMob.AD_POSITION.BOTTOM_CENTER options in AdMob

**Comment by shprink** [comment]
Interesting. If you disable this plugin does it work correctly?

**Comment by Shinzah** [comment]
Yes, if i disable the NativeTransition plugin, it work fine.
If using the AdMob.AD_POSITION.TOP_CENTER, all work fine. (The add stay in top, and the page content is correct)

**Comment by shprink** [comment]
@EddyVerbruggen do you think it is related to this plugin? Have you tested your plugin with admobs? 

Thanks

**Comment by EddyVerbruggen** [comment]
Nope, I can't possibly test all plugins with all other plugins ;)
But I can imagine the fact that the webview is pushed up by admob will interfere with the transitions plugin.

Can you post how you're using the plugin? Slide with fixedPixelsBottom perhaps?

**Comment by Shinzah** [comment]
I only use 
```javascript
$ionicNativeTransitions.enable(true);
```
Without any other configuration.

And for admobs :
```javascript
$cordovaGoogleAds.createBanner({
                        adId: banner_id,
                        position: AdMob.AD_POSITION.BOTTOM_CENTER,
                        autoShow: false
                    }).then(function() {
                        _bannerCreated = true;
                        $cordovaGoogleAds.showBanner(AdMob.AD_POSITION.BOTTOM_CENTER);
                    });
```

**Comment by degroundshaker** [comment]
I came up with the same issue. 

I solved it by setting **overlap** parameter to **_true_**

**Comment by shprink** [comment]
Awesome @Shinzah what to try to see if it solves your problem? If it does please close this ticket

**Comment by degroundshaker** [comment]
@shprink For me, it did 100.1% :)

**Comment by Shinzah** [comment]
Hi,
Indeed it works with native transition, but the problem is the "overlap" ... which ... well, overlap the bottom of the screen.
And because of the device size, the height is not always the same, so i can't add a padding bottom to fix that :(
But thanks for the trick !

**Comment by rodrigograca31** [comment]
I can confirm this issue!
I've been using the admob plugin `cordova-plugin-admobpro` (version: 2.18.0) and it worked fine until I added this plugin.... It went from the bottom (`position: AdMob.AD_POSITION.BOTTOM_CENTER`) to the top when I added this plugin....

Using `overlap: true` may work but it's not the best solution.... Because from then on you have to have some padding on your views or content will be overlapped by admob....

Also: I've removed this plugin and admob is not showing now....
So far I've tried `cordova platform rm android` and add it back but admob is still not showing...
